ql/src/test/org/apache/hadoop/hive/ql/metadata/TestHiveMetaStoreChecker.java

```diff
@@ -82,6 +84,11 @@ public void setUp() throws Exception {
     msc = new HiveMetaStoreClient(hive.getConf());
     checker = new HiveMetaStoreChecker(msc, hive.getConf());
 
+    hive.getConf().setVar(HiveConf.ConfVars.HIVE_AUTHORIZATION_MANAGER,
+        "org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdHiveAuthorizerFactory");
+    HiveConf.setBoolVar(hive.getConf(), HiveConf.ConfVars.HIVE_SUPPORT_CONCURRENCY, false);
+    SessionState.start(hive.getConf());
```

Comment:
Why are these changes are needed?

Reply:
All tests here deal with transactional table, but doesn't initiate SessionState. So, this results in Nullpointer, when we try to retrieve transaction manager from SessionState, in order to derive ValidWriteIdList.